data/json/items/comestibles/med.json

```
@@ -1331,11 +1331,15 @@
   },
   {
     "id": "disincotton_ball",
-    "copy-from": "cotton_ball",
     "type": "COMESTIBLE",
     "name": "disinfectant soaked cotton balls",
     "name_plural": "disinfectant soaked cotton balls",
     "description": "Fluffy balls of clean white cotton.  Now soaked with disinfectant, they are somewhat useful to disinfect a wound.",
+    "weight": 25,
+    "//": "Can't copy-from cotton_ball, breaks volume",
+    "volume": 0.125,
```

I don't think this works.
It doesn't matter though, because volume should be in strings.
```suggestion
    "volume": "31 ml",
```